docs/rewards/cell-capacity.mdx

```diff
@@ -10,12 +10,19 @@ slug: /project/cell-capacity
 
 Each cell has a predefined cell capacity that depends on its geospatial characteristics. 
 The cell's capacity is the maximum number of stations that may receive rewards in a specific cell. 
-If 'N' weather stations are rewardable in a cell with a capacity CC during a specific day, if N>CC, then only the first N stations (ordered first by their rewards score defined below, and then by seniority) will receive rewards.
+If 'N' weather stations are rewardable in a cell with a capacity CC during a specific day, if N>CC, then only the first N stations 
+(ordered first by their rewards score defined below, and then by seniority) will receive rewards.
+The Cell's Capacity is a parameter that is used to define the maximum number of stations that may receive rewards in a specific cell.
 
 ### Last Claim Time (LCT): 
 Claim time is a unix timestamp (integer).
 Last Claim Time is set to the timestamp of the last claim of the station. It is used as a seniority indicator ( LCT1<LCT2 then station 1 is more senior than station 2).
 
+### Cell Position 
+As long as the station's position is above the maximum capacity it will be rewarded, whereas getting below this threshold will result in no rewards.
```

Comment:
It's hard to understand what is "above maximum capacity" and what is "below maximum capacity". I would think it means the opposite thing that what we want to say here. Let's rephrase it to:

> As long as the station's position is inside the maximum capacity range it will be rewarded, whereas getting out of that range will result in no rewards.

What do you think?

Reply:
Good thought